[2021-05-12 11:10:58,718] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: status_test.task4 2021-05-12T10:39:48.267484+00:00 [queued]>
[2021-05-12 11:10:58,726] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: status_test.task4 2021-05-12T10:39:48.267484+00:00 [queued]>
[2021-05-12 11:10:58,726] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 11:10:58,726] {taskinstance.py:1069} INFO - Starting attempt 7 of 8
[2021-05-12 11:10:58,726] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 11:10:58,861] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): task4> on 2021-05-12T10:39:48.267484+00:00
[2021-05-12 11:10:58,863] {standard_task_runner.py:52} INFO - Started process 2200572 to run task
[2021-05-12 11:10:58,868] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'status_test', 'task4', '2021-05-12T10:39:48.267484+00:00', '--job-id', '96', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task_status.py', '--cfg-path', '/tmp/tmpt61c6nol', '--error-file', '/tmp/tmpz1522ykf']
[2021-05-12 11:10:58,869] {standard_task_runner.py:77} INFO - Job 96: Subtask task4
[2021-05-12 11:10:59,006] {logging_mixin.py:104} INFO - Running <TaskInstance: status_test.task4 2021-05-12T10:39:48.267484+00:00 [running]> on host sdn-k8s-b2-3-3.maas
[2021-05-12 11:10:59,122] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=davidwu5858@gmail.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=status_test
AIRFLOW_CTX_TASK_ID=task4
AIRFLOW_CTX_EXECUTION_DATE=2021-05-12T10:39:48.267484+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-12T10:39:48.267484+00:00
[2021-05-12 11:10:59,122] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-05-12 11:10:59,123] {bash.py:158} INFO - Running command: airflow tasks clear -f status_test\ sleep 5 \ yes
[2021-05-12 11:10:59,129] {bash.py:169} INFO - Output:
[2021-05-12 11:10:59,701] {bash.py:173} INFO - usage: airflow [-h] GROUP_OR_COMMAND ...
[2021-05-12 11:10:59,701] {bash.py:173} INFO - 
[2021-05-12 11:10:59,701] {bash.py:173} INFO - positional arguments:
[2021-05-12 11:10:59,701] {bash.py:173} INFO -   GROUP_OR_COMMAND
[2021-05-12 11:10:59,701] {bash.py:173} INFO - 
[2021-05-12 11:10:59,702] {bash.py:173} INFO -     Groups:
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       celery         Celery components
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       config         View configuration
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       connections    Manage connections
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       dags           Manage DAGs
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       db             Database operations
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       kubernetes     Tools to help run the KubernetesExecutor
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       pools          Manage pools
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       providers      Display providers
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       roles          Manage roles
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       tasks          Manage tasks
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       users          Manage users
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       variables      Manage variables
[2021-05-12 11:10:59,702] {bash.py:173} INFO - 
[2021-05-12 11:10:59,702] {bash.py:173} INFO -     Commands:
[2021-05-12 11:10:59,702] {bash.py:173} INFO -       cheat-sheet    Display cheat sheet
[2021-05-12 11:10:59,703] {bash.py:173} INFO -       info           Show information about current Airflow and environment
[2021-05-12 11:10:59,703] {bash.py:173} INFO -       kerberos       Start a kerberos ticket renewer
[2021-05-12 11:10:59,703] {bash.py:173} INFO -       plugins        Dump information about loaded plugins
[2021-05-12 11:10:59,703] {bash.py:173} INFO -       rotate-fernet-key
[2021-05-12 11:10:59,703] {bash.py:173} INFO -                      Rotate encrypted connection credentials and variables
[2021-05-12 11:10:59,703] {bash.py:173} INFO -       scheduler      Start a scheduler instance
[2021-05-12 11:10:59,703] {bash.py:173} INFO -       sync-perm      Update permissions for existing roles and DAGs
[2021-05-12 11:10:59,703] {bash.py:173} INFO -       version        Show the version
[2021-05-12 11:10:59,703] {bash.py:173} INFO -       webserver      Start a Airflow webserver instance
[2021-05-12 11:10:59,703] {bash.py:173} INFO - 
[2021-05-12 11:10:59,703] {bash.py:173} INFO - optional arguments:
[2021-05-12 11:10:59,703] {bash.py:173} INFO -   -h, --help         show this help message and exit
[2021-05-12 11:10:59,703] {bash.py:173} INFO - 
[2021-05-12 11:10:59,703] {bash.py:173} INFO - airflow command error: unrecognized arguments: 5  yes, see help above.
[2021-05-12 11:10:59,800] {bash.py:177} INFO - Command exited with return code 2
[2021-05-12 11:10:59,807] {taskinstance.py:1482} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/venv/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1138, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/ubuntu/venv/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1311, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/ubuntu/venv/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1341, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/ubuntu/venv/lib/python3.8/site-packages/airflow/operators/bash.py", line 180, in execute
    raise AirflowException('Bash command failed. The command returned a non-zero exit code.')
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code.
[2021-05-12 11:10:59,809] {taskinstance.py:1525} INFO - Marking task as UP_FOR_RETRY. dag_id=status_test, task_id=task4, execution_date=20210512T103948, start_date=20210512T111058, end_date=20210512T111059
[2021-05-12 11:10:59,932] {local_task_job.py:146} INFO - Task exited with return code 1
